File: JavaAssignment/LoanAssignment/loan/PersonalLoan.java
package loan;
public class PersonalLoan extends Loan{
    
    public PersonalLoan(double principle,int period){
        super(principle,period);
    }

    @Override
    public double getRate() {
        return super.getPrinciple()<=500000?0.15:0.16;
    }

}

File: DotNet_master/DotNetAssignments/Loan2/loanapp/PersonalLoan.cs
namespace loanapp{


public sealed class PersonalLoan : Loan,ITaxable{
     internal PersonalLoan(long principle,int period):base(principle,period){}

         public override float GetRate(){
            return (principle<=500000)?0.15f:0.16f;
        }  

        public double getTax(double amount){
            return amount*=1.1;
        } 
    }

}